app/controllers/ingestibles_controller.rb

```diff
@@ -4,8 +4,8 @@ class IngestiblesController < ApplicationController
   include LockIngestibleConcern
 
   before_action { |c| c.require_editor('edit_catalog') }
```

Comment:
this is missing in the new separate controller

Reply:
Oops, thank you, overlooked this. Fixed now